docs(format-error): add an explanation that the original error wrapped in GraphQLError

denis-bel opened the pull request:
Hi all! I was working with error handling in my project and wasted a lot of time because I couldn't figure out why I can identify my original error with "instanceof". It wasn't until after some research on StackOverflow that I realized that the original error was wrapped in a GraphQLError. Can we add this detail to the documentation? Thanks, everyone in advance

apollo-cla commented:
@denis-bel: Thank you for submitting a pull request! Before we can merge it, you'll need to sign the Apollo Contributor License Agreement here: https://contribute.apollographql.com/

denis-bel commented:
> For what it's worth, this is mentioned in errors.mdx, if you search for `unwrapResolverError`. Which isn't to say that I disagree with your additions in this place, but I wonder if adding an appropriate link to your changes or a reference to that function or something might also be helpful?
> 
> (I think grammar-wise "thrown by a resolver" reads better than "thrown by resolver", too.)

I think adding a link or reference is a good idea. And I agree that "thrown by a resolver" is much better

denis-bel commented:
@glasser I updated "thrown by resolver" to "thrown by resolver" and added a reference in "apollo-server.mdx" file. I was also thinking about adding a link to the `errors.mdx` file, but I came across that there is no section (marked with #) to make a link to the `unwrapResolverError` function. I think it shouldn't be a problem if people read the whole section

trevor-scheer reviewed (approved):
Thanks!
